### DISCUSSION TITLE
Expo version warning, do I just ignore them?

### DISCUSSION THREAD

**Discussion by chen-rn** [nandorojo/solito#278] [Q&A] [answered]
<img width="870" alt="Screenshot 2022-12-11 at 3 19 43 PM" src="https://user-images.githubusercontent.com/36214945/206934859-fb86f007-e972-44bc-b78e-74e3497fd400.png">

**Comment by nandorojo** [comment, marked as answer]
you can ignore the react ones, but i would increment RN

**Reply by chen-rn** [reply]
I see! Also, does solito work with expo 37?

**Reply by nandorojo** [reply]
doubt it. do you mean 47?

**Reply by chen-rn** [reply]
gotcha! yes that's what i meant!
